src/ert/run_models/run_model.py

```diff
@@ -807,7 +807,7 @@ def _evaluate_and_postprocess(
         )
         logger.info(
             f"Experiment run ended with number of realizations failing: "
-            f"{self.ensemble_size - num_successful_realizations}"
+            f"{len(starting_realizations) - num_successful_realizations}"
```

Comment:
The similar log message on line 803 is perhaps equally wrong? Maybe the log (on line 803) should state both the size of the ensemble, and the size of the starting realizations?

Reply:
Have updated a new proposal